NGitLab/Models/IssueCreate.cs

```diff
@@ -33,5 +33,11 @@ public class IssueCreate
         [JsonPropertyName("due_date")]
         [JsonConverter(typeof(DateOnlyConverter))]
         public DateTime? DueDate;
+
+        [JsonPropertyName("epic_id")]
+        public int? EpicId;
+
+        [JsonPropertyName("epic_iid")]
+        public int? EpicIId;
```

Comment:
Don't add this field, as it's been deprecated by GitLab and will eventually be removed. See [here](https://docs.gitlab.com/ee/api/issues.html#new-issue).

Reply:
yes I know, but we need this field in older gitlab version like we used at Ubisoft. Epic_id needs extra permission